### PR TITLE
`shootLength` for other turrets that aren't `PointDefenceTurret`s

### PR DESCRIPTION
**Pull request by MEEPofFaith**
https://user-images.githubusercontent.com/54301439/104276080-fb9a9800-5458-11eb-81a2-57cb75a0d5cf.mp4

### CODE REVIEW

**Comment by Sh1penfire** [comment]
Anuken, please merge this.
It allows hjson/json modders to change the shooting position for turrets without script.
js and java modders will benefit from this as they won't have to write scripts for this certain issue.

**Comment by MEEPofFaith** [comment]
~~Team Avant don't hurt me for using Extinction as an example.~~

**Review by sk7725 — APPROVED** [review]

File: core/src/mindustry/world/blocks/defense/turrets/TractorBeamTurret.java [hunk 1]
```diff
@@ -24,8 +24,9 @@
     public @Load("block-@size") TextureRegion baseRegion;
     public @Load("@-laser") TextureRegion laser;
     public @Load("@-laser-end") TextureRegion laserEnd;
-
+
```
Comment (joshuaptfan) [inline core/src/mindustry/world/blocks/defense/turrets/TractorBeamTurret.java:27]: You accidentally added unnecessary spaces here.

Comment (MEEPofFaith) [reply core/src/mindustry/world/blocks/defense/turrets/TractorBeamTurret.java:27]: meh

**Pull request merged**